src/query/vacuum.c

```diff
@@ -7428,10 +7435,10 @@ vacuum_notify_server_shutdown (void)
  *
  * return : Global oldest active MVCCID.
  */
-VACUUM_LOG_BLOCKID
+MVCCID
```

Comment:
We checked logpb_flush_all_append_pages, and we removed the logpb_Rmutex_log_pb because between demote and promote there are other active mutexes: writer_info->flush_start_mutex, flush_info->flush_mutex, writer_info->flush_wait_mutex, writer_info->flush_end_mutex, so there will be only one thread entering those critical functions (like logpb_set_dirty). If you want, I can do for debug reasons some checks to see that we have at least one mutex.